hosts.example

```diff
@@ -105,58 +101,80 @@
 # mgmt
 # bridge
 
-# sp_node_roles='client'	# Default
-# sp_node_roles='server,iscsi'	# Another Example
+# sp_node_roles='client'	        # Default
+# sp_node_roles='server,iscsi'	    # Another Example
 
 # List of arguments to pass to storpool_cg conf
 # NOTE: See 'storpool_cg conf -h' for all available arguments
-#
+
 # Examples:
 # sp_cg_conf_extra='CORES=2 NUMA_OVERFLOW=1 SP_COMMON_LIMIT=2G SP_ALLOC_LIMIT=2G'
 # sp_cg_conf_extra='SYSTEM_LIMIT=4G USER_LIMIT=2G MACHINE_LIMIT=4G'
-#
-# Default if client+server is defined in sp_node_roles
-# sp_cg_conf_extra='CONVERGED=1'
-#
+
 # Default otherwise
 # sp_cg_conf_extra=''
+
+[storpool:children]
+storpool_servers
+
+[storpool_servers]
+
+
+[storpool_servers:vars]
+# Drive selection section
+#   - Values can be obtained by running 'lsblk -do NAME,SIZE,MODEL' on the nodes
+#   - Devices matching _any_ of the conditions below will get initialized
+#   - Larger NVME drives will be split into equally-sized partitions smaller than 4T
+#   - A matched device, already containing partitions, will trigger an error
+#     remove partitions manually or override with sp_drive_erase=True
 #
+# CAUTION: The drive_init procedure is not fully implemented yet
+#   - No RAID controller detection, and private case scenarios
+
+# Minimal size of SATA drives (SSD and HDD) below which drives won't be discovered to be used by StorPool
+# sp_min_disk_size='960GB'
+
+# Maximal size of SATA drives (SSD and HDD) above which drives won't be discovered to be used by StorPool
+# sp_max_disk_size='14TB'
+
+# Minimal size of NVMe drives below which drives won't be discovered to be used by StorPool
+# sp_min_nvme_size='960GB'
+
+# Minimal size of PMEM drives (SSD and HDD) below which drives won't be discovered to be used by StorPool
+# sp_min_pmem_size='1GB'
+
+# Whether to run the discover phase of disk_init_helper. Set to false to use a pre-generated JSON file to be
+# used for disk initialization
+# sp_discover_disk_run=True
+
+# Disk glob pattern to discover disks by. Specify manufacturer model name, serial number.
+# sp_discover_disk_pattern=''
```

Comment:
is there a default, or by default it doesn't match anything?

also, an example would be great

Reply:
I've added a link to the user guide that shows how pattern matching works